src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -170,6 +170,8 @@
     <Compile Include="System\Resources\ResourceSet.cs" />
     <Compile Include="System\Runtime\InteropServices\CriticalHandle.cs" />
     <Compile Include="System\Text\CodePageDataItem.cs" />
+    <Compile Include="System\Threading\ClrThreadPool.Unix.cs" />
+    <Compile Include="System\Threading\ClrThreadPool.HillClimbing.Unix.cs" />
```

Comment:
Personally, I would like to see the code structured in such a way that would be easy to enable ClrThreadPool on Windows too if it is needed (for example, for experiential/perf comparison purposes).